dask_jobqueue/core.py

```diff
@@ -521,7 +536,26 @@ def __init__(
         job_kwargs["config_name"] = config_name
         job_kwargs["interface"] = interface
         job_kwargs["protocol"] = protocol
-        job_kwargs["security"] = security
+        job_kwargs["security"] = copy.copy(security)
```

Comment:
Adding it at this point changes the object only for the job. I did it centrally though to not generate one temp file per certificate and job

Reply:
Why do we need copy?

Reply:
because I modify this pass by reference item: I am inlining the certs: see #520. IMHO it confuses people if their object is changed.